zipline/pipeline/loaders/blaze.py

```diff
@@ -648,11 +648,31 @@ def overwrite_from_dates(asof, dense_dates, sparse_dates, asset_idx, value):
     -------
     overwrite : Float64Overwrite
         The overwrite that will apply the new value to the data.
+
+    Notes
+    -----
+    The overwrite is forward filled in the sense that it applies from the index
+    of the ``asof`` in the ``dense_dates`` to the index of the next date in the
```

Comment:
The docstring up above this says "based on the asof date of the delta, the dense_dates, and the dense_dates".